package.json

```diff
@@ -1,16 +1,16 @@
 {
 	"name": "got-scraping",
-	"version": "1.0.4",
+	"version": "1.0.5",
 	"description": "HTTP client made for scraping based on got.",
 	"main": "src/index.js",
 	"files": [
 		"src"
 	],
 	"dependencies": {
-		"header-generator": "^0.0.1-beta.1",
 		"got": "^11.8.1",
+		"header-generator": "^0.0.1-beta.1",
```

Comment:
We could use the 1.0.0 version here. It does not have any breaking changes from v0.